python/cudf/cudf/_lib/cpp/io/timezone.pxd

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020-2023, NVIDIA CORPORATION.
+
+from libcpp cimport bool
+from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string
+
+from cudf._lib.cpp.table.table cimport table
+
+
+# TODO: this is vendored from Cython 3.0
```

Comment:
We ought to create a tracking issue for cleanup work to do after Cython 3 is released. Also make sure we are cleared to vendor Cython code with licensing/etc.

Reply:
https://github.com/rapidsai/cudf/issues/13114